### PR TITLE
feat: add standard platform checks files

### PR DESCRIPTION
**Pull request by lisastedmanfalls-gametime**
Hello, I would like to make Gametime a better place by contributing the following code:

## Feature/bug description

Since platform has inherited this repo, we want to add our standard checks to it.

## This is how I decided to implement/fix it

- Add precommit, pr-template, codeowner of platform, and update makefile

## JIRA link

https://gametime.atlassian.net/jira/software/c/projects/GT/boards/65?selectedIssue=PLT-447

## What does this change affect? (What can this break?)
Describe the critical path(s) affected as well as the related component and system functions.

Just affects this repo's checks

## How has this been tested

This PR

## Observability

Before opening a PR consider whether you have added sufficient observability, do you need to add any datadog additional metrics or spans?

- [ ] Do your metrics follow the naming conventions?

### CODE REVIEW

**Comment by lisastedmanfalls-gametime** [comment]
I think this repo would need to be private to run the workflows

**Comment by lisastedmanfalls-gametime** [comment]
Cannot add workflows just adding what I can for now

File: Makefile [hunk 1]
```diff
@@ -1,20 +1,23 @@
-OUTPUT := bin/${current_dir}
+OUTPUT = bin/ecs-deploy
+BUILD_DATE=`date +%FT%T%z`
+GIT_HASH=`git rev-parse HEAD`
+LDFLAGS=-ldflags="-X github.com/gametimesf/log.builtAt=${BUILD_DATE} -X github.com/gametimesf/log.commitHash=${GIT_HASH}"
```
Comment (waitstime) [inline Makefile:4]: can remove this line as we don't use our log package here

**Review by waitstime — APPROVED** [review]
just one comment

File: Makefile (hunk 1 above)
Comment (lisastedmanfalls-gametime) [reply Makefile]: done. removed that and the ohter code associated with that

**Review by particledecay — APPROVED** [review]

**Pull request merged**